pkg/api/apiloader.go

```diff
@@ -35,6 +35,40 @@ func (a *Apiloader) LoadContainerServiceFromFile(jsonFile string, validate, isUp
 	return a.DeserializeContainerService(contents, validate, isUpdate, existingContainerService)
 }
 
+// LoadDefaultContainerServiceProperties loads the default API model
+func LoadDefaultContainerServiceProperties() (TypeMeta, *vlabs.Properties) {
+	return TypeMeta{APIVersion: vlabs.APIVersion}, &vlabs.Properties{
+		OrchestratorProfile: &vlabs.OrchestratorProfile{
+			OrchestratorType:    "kubernetes",
+			OrchestratorVersion: "1.10",
+			KubernetesConfig: &vlabs.KubernetesConfig{
+				NetworkPlugin:      "azure",
```

Comment:
I think we should network policy by default. @jackfrancis can we use Calico with AzureCNI?

Reply:
No, though `"NetworkPolicy": "azure"` (i.e., Azure NetworkPolicy implementation) is now possible. @saiyan86 can speak to how experimental that feature is at the moment.